app/src/main/java/com/iic/shopingo/ui/OnboardingActivity.java

```diff
@@ -39,6 +45,14 @@ public void call(Session session, SessionState sessionState, Exception e) {
   @Override
   protected void onResume() {
     super.onResume();
+
+    // For scenarios where the main activity is launched and user session is not null, the session state change notification
```

Comment:
You also get `onStart`. `onResume` is only relevant when you have a dialog on top of your activity.
Anyway, isn't it enough to do all of this in onCreate? What's the use case here?